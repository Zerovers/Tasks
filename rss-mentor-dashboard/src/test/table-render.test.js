import React from 'react';
import { shallow } from 'enzyme';
import Table from '../components/table';

it('should render a table', () => {
  const mentorName = 'MikhailLU';
  const table = shallow(<Table mentorName={mentorName}/>);
  expect(table).toMatchSnapshot();
});

