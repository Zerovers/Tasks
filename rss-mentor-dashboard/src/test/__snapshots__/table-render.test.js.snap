// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a table 1`] = `
<table>
  <thead>
    <tr>
      <th />
      <th
        key="elusiveway"
        style={
          Object {
            "background": "#4095bf",
          }
        }
      >
        <a
          href="https://github.com/elusiveway"
        >
          elusiveway
        </a>
      </th>
      <th
        key="cirillius"
        style={
          Object {
            "background": "#4095bf",
          }
        }
      >
        <a
          href="https://github.com/cirillius"
        >
          cirillius
        </a>
      </th>
      <th
        key="eugenepochenkov"
        style={
          Object {
            "background": "#4095bf",
          }
        }
      >
        <a
          href="https://github.com/eugenepochenkov"
        >
          eugenepochenkov
        </a>
      </th>
      <th
        key="lx-931203"
        style={
          Object {
            "background": "#4095bf",
          }
        }
      >
        <a
          href="https://github.com/lx-931203"
        >
          lx-931203
        </a>
      </th>
      <th
        key="astravukh"
        style={
          Object {
            "background": "#4095bf",
          }
        }
      >
        <a
          href="https://github.com/astravukh"
        >
          astravukh
        </a>
      </th>
      <th
        key="zetorby"
        style={
          Object {
            "background": "#4095bf",
          }
        }
      >
        <a
          href="https://github.com/zetorby"
        >
          zetorby
        </a>
      </th>
    </tr>
  </thead>
  <tbody>
    <tr
      key="Code Jam \\"CV\\""
    >
      <td
        style={
          Object {
            "background": "#2db91a",
          }
        }
      >
        <a
          href="https://github.com/rolling-scopes-school/tasks/blob/2018-Q3/tasks/codejam-cv.md"
        >
          Code Jam "CV"
        </a>
      </td>
      <td
        key="elusiveway"
        style={
          Object {
            "background": "#721d1d",
          }
        }
      />
      <td
        key="cirillius"
        style={
          Object {
            "background": "#721d1d",
          }
        }
      />
      <td
        key="eugenepochenkov"
        style={
          Object {
            "background": "#721d1d",
          }
        }
      />
      <td
        key="lx-931203"
        style={
          Object {
            "background": "#721d1d",
          }
        }
      />
      <td
        key="astravukh"
        style={
          Object {
            "background": "#721d1d",
          }
        }
      />
      <td
        key="zetorby"
        style={
          Object {
            "background": "#721d1d",
          }
        }
      />
    </tr>
    <tr
      key="Code Jam \\"CoreJS\\""
    >
      <td
        style={
          Object {
            "background": "#2db91a",
          }
        }
      >
        <a
          href="https://github.com/rolling-scopes-school/tasks/blob/2018-Q3/tasks/codejam-corejs.md"
        >
          Code Jam "CoreJS"
        </a>
      </td>
      <td
        key="elusiveway"
        style={
          Object {
            "background": "#721d1d",
          }
        }
      />
      <td
        key="cirillius"
        style={
          Object {
            "background": "#721d1d",
          }
        }
      />
      <td
        key="eugenepochenkov"
        style={
          Object {
            "background": "#721d1d",
          }
        }
      />
      <td
        key="lx-931203"
        style={
          Object {
            "background": "#721d1d",
          }
        }
      />
      <td
        key="astravukh"
        style={
          Object {
            "background": "#721d1d",
          }
        }
      />
      <td
        key="zetorby"
        style={
          Object {
            "background": "#721d1d",
          }
        }
      />
    </tr>
    <tr
      key="Code Jam \\"DOM, DOM Events\\""
    >
      <td
        style={
          Object {
            "background": "#2db91a",
          }
        }
      >
        <a
          href="https://github.com/rolling-scopes-school/tasks/blob/2018-Q3/tasks/codejam-notification.md"
        >
          Code Jam "DOM, DOM Events"
        </a>
      </td>
      <td
        key="elusiveway"
        style={
          Object {
            "background": "#721d1d",
          }
        }
      />
      <td
        key="cirillius"
        style={
          Object {
            "background": "#721d1d",
          }
        }
      />
      <td
        key="eugenepochenkov"
        style={
          Object {
            "background": "#2db91a",
          }
        }
      />
      <td
        key="lx-931203"
        style={
          Object {
            "background": "#721d1d",
          }
        }
      />
      <td
        key="astravukh"
        style={
          Object {
            "background": "#721d1d",
          }
        }
      />
      <td
        key="zetorby"
        style={
          Object {
            "background": "#2db91a",
          }
        }
      />
    </tr>
    <tr
      key="Markup #1"
    >
      <td
        style={
          Object {
            "background": "#f06d6d",
          }
        }
      >
        <a
          href="https://github.com/rolling-scopes-school/tasks/blob/2018-Q3/tasks/markup-2018q3.md"
        >
          Markup #1
        </a>
      </td>
      <td
        key="elusiveway"
        style={
          Object {
            "background": "#f06d6d",
          }
        }
      />
      <td
        key="cirillius"
        style={
          Object {
            "background": "#f06d6d",
          }
        }
      />
      <td
        key="eugenepochenkov"
        style={
          Object {
            "background": "#2db91a",
          }
        }
      />
      <td
        key="lx-931203"
        style={
          Object {
            "background": "#f06d6d",
          }
        }
      />
      <td
        key="astravukh"
        style={
          Object {
            "background": "#f06d6d",
          }
        }
      />
      <td
        key="zetorby"
        style={
          Object {
            "background": "#2db91a",
          }
        }
      />
    </tr>
    <tr
      key="RS Activist"
    >
      <td
        style={
          Object {
            "background": "#ecce6c",
          }
        }
      >
        <a
          href="https://github.com/rolling-scopes-school/tasks/blob/2018-Q3/tasks/rs-school-activist.md"
        >
          RS Activist
        </a>
      </td>
      <td
        key="elusiveway"
        style={
          Object {
            "background": "#ecce6c",
          }
        }
      />
      <td
        key="cirillius"
        style={
          Object {
            "background": "#ecce6c",
          }
        }
      />
      <td
        key="eugenepochenkov"
        style={
          Object {
            "background": "#ecce6c",
          }
        }
      />
      <td
        key="lx-931203"
        style={
          Object {
            "background": "#ecce6c",
          }
        }
      />
      <td
        key="astravukh"
        style={
          Object {
            "background": "#ecce6c",
          }
        }
      />
      <td
        key="zetorby"
        style={
          Object {
            "background": "#ecce6c",
          }
        }
      />
    </tr>
    <tr
      key="YouTube"
    >
      <td
        style={
          Object {
            "background": "#f06d6d",
          }
        }
      >
        <a
          href="https://github.com/rolling-scopes-school/tasks/blob/2018-Q3/tasks/youtube.md"
        >
          YouTube
        </a>
      </td>
      <td
        key="elusiveway"
        style={
          Object {
            "background": "#f06d6d",
          }
        }
      />
      <td
        key="cirillius"
        style={
          Object {
            "background": "#f06d6d",
          }
        }
      />
      <td
        key="eugenepochenkov"
        style={
          Object {
            "background": "#f06d6d",
          }
        }
      />
      <td
        key="lx-931203"
        style={
          Object {
            "background": "#f06d6d",
          }
        }
      />
      <td
        key="astravukh"
        style={
          Object {
            "background": "#f06d6d",
          }
        }
      />
      <td
        key="zetorby"
        style={
          Object {
            "background": "#f06d6d",
          }
        }
      />
    </tr>
    <tr
      key="Code Jam \\"Scoreboard\\""
    >
      <td
        style={
          Object {
            "background": "#f06d6d",
          }
        }
      >
        <a
          href="https://github.com/rolling-scopes-school/tasks/blob/2018-Q3/tasks/codejam-scoreboard.md"
        >
          Code Jam "Scoreboard"
        </a>
      </td>
      <td
        key="elusiveway"
        style={
          Object {
            "background": "#f06d6d",
          }
        }
      />
      <td
        key="cirillius"
        style={
          Object {
            "background": "#f06d6d",
          }
        }
      />
      <td
        key="eugenepochenkov"
        style={
          Object {
            "background": "#f06d6d",
          }
        }
      />
      <td
        key="lx-931203"
        style={
          Object {
            "background": "#f06d6d",
          }
        }
      />
      <td
        key="astravukh"
        style={
          Object {
            "background": "#f06d6d",
          }
        }
      />
      <td
        key="zetorby"
        style={
          Object {
            "background": "#f06d6d",
          }
        }
      />
    </tr>
    <tr
      key="Game"
    >
      <td
        style={
          Object {
            "background": "#ecce6c",
          }
        }
      >
        <a
          href="https://github.com/rolling-scopes-school/tasks/blob/2018-Q3/tasks/game.md"
        >
          Game
        </a>
      </td>
      <td
        key="elusiveway"
        style={
          Object {
            "background": "#ecce6c",
          }
        }
      />
      <td
        key="cirillius"
        style={
          Object {
            "background": "#ecce6c",
          }
        }
      />
      <td
        key="eugenepochenkov"
        style={
          Object {
            "background": "#ecce6c",
          }
        }
      />
      <td
        key="lx-931203"
        style={
          Object {
            "background": "#ecce6c",
          }
        }
      />
      <td
        key="astravukh"
        style={
          Object {
            "background": "#ecce6c",
          }
        }
      />
      <td
        key="zetorby"
        style={
          Object {
            "background": "#ecce6c",
          }
        }
      />
    </tr>
    <tr
      key="Presentation"
    >
      <td
        style={
          Object {
            "background": "#f06d6d",
          }
        }
      >
        <a
          href=""
        >
          Presentation
        </a>
      </td>
      <td
        key="elusiveway"
        style={
          Object {
            "background": "#f06d6d",
          }
        }
      />
      <td
        key="cirillius"
        style={
          Object {
            "background": "#f06d6d",
          }
        }
      />
      <td
        key="eugenepochenkov"
        style={
          Object {
            "background": "#f06d6d",
          }
        }
      />
      <td
        key="lx-931203"
        style={
          Object {
            "background": "#f06d6d",
          }
        }
      />
      <td
        key="astravukh"
        style={
          Object {
            "background": "#f06d6d",
          }
        }
      />
      <td
        key="zetorby"
        style={
          Object {
            "background": "#f06d6d",
          }
        }
      />
    </tr>
    <tr
      key="Course work"
    >
      <td
        style={
          Object {
            "background": "",
          }
        }
      >
        <a>
          Course work
        </a>
      </td>
      <td
        key="elusiveway"
        style={
          Object {
            "background": "#7a7979",
          }
        }
      />
      <td
        key="cirillius"
        style={
          Object {
            "background": "#7a7979",
          }
        }
      />
      <td
        key="eugenepochenkov"
        style={
          Object {
            "background": "#7a7979",
          }
        }
      />
      <td
        key="lx-931203"
        style={
          Object {
            "background": "#7a7979",
          }
        }
      />
      <td
        key="astravukh"
        style={
          Object {
            "background": "#7a7979",
          }
        }
      />
      <td
        key="zetorby"
        style={
          Object {
            "background": "#7a7979",
          }
        }
      />
    </tr>
  </tbody>
</table>
`;
